docs/release-notes/mongoid-7.4.txt

```diff
@@ -428,3 +436,95 @@ distinct methods will not be demongoized. For example:
 
   The value of ``Mongoid.legacy_pluck_distinct`` option will change to ``false``
   by default in Mongoid 8.0.
+
+Localized Fields with ``pluck`` and ``distinct``
+````````````````````````````````````````````````
+Mongoid 7.4 with the ``Mongoid.legacy_pluck_distinct`` option set to ``false``
+changes the behavior of using ``pluck`` and ``distinct`` with localized fields.
+Now when inputting a localized field into these functions, the translation for
```

Comment:
I would say "when retrieving a localized field". Also Ruby doesn't have functions, it has methods.

Reply:
done